nic_operator_helm/nic_operator_helm_ci_start.sh

```diff
@@ -98,7 +98,14 @@ function deploy_operator {
 
     # Install charts from 3rd-party repositories. Errors for empty reposetories won't affect installation.
     cd deployment/network-operator
-    yq '.dependencies[] | "\(.name) \(.repository)"'  Chart.yaml |  tr -d '"' | xargs -t  -L 1 helm repo add
+    local charts=$(yq r Chart.yaml dependencies  -l)
```

Comment:
nit: if Chart.yaml have not `dependencies` block `seq` command below will fail. This will not block execution, but error will be printed

Reply:
Thanks, I didn't think about it. IMO, in this particular case it's not an issue because we aren't going to drop network-operator dependencies soon